beacon-chain/state/state-native/getters_participation.go

```diff
@@ -36,6 +39,50 @@ func (b *BeaconState) PreviousEpochParticipation() ([]byte, error) {
 	return b.previousEpochParticipationVal(), nil
 }
 
+// UnrealizedCheckpointBalances returns the total balances: active, target attested in
+// current epoch and target attested in previous epoch. This function is used to
+// compute the "unrealized justification" that a synced Beacon Block will have.
+func (b *BeaconState) UnrealizedCheckpointBalances() (uint64, uint64, uint64, error) {
```

Comment:
If I'm not mistaken, this function should not be supported for `b.version == version.Phase0`

Reply:
Oh thanks, I added an explicit error in v1 but didn't even think about phase0 checks in native state